components/src/stories/PromptModal.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { PropmptModal } from "../Modal/variants/PromptModal/PromptModal";
+
+const meta: Meta<typeof PropmptModal> = {
+  title: "Components/PropmptModal",
+  component: PropmptModal,
+};
+export default meta;
+
+type Story = StoryObj<typeof PropmptModal>;
+
+export const Default: Story = {
+  args: {
+    isOpen: true,
+    onClose: () => {},
+  },
+  render: (args) => (
+    <PropmptModal
+      {...args}
+      title="Prompt 모달"
+      content="Prompt 모달 내용입니다."
+      placeholder="Prompt placeholder"
+    />
+  ),
+};
```

Comment:
현재 args 를 할당하는 부분 하단에 props 가 고정적으로 할당되고 있기 때문에 스토리북 controls 패널에서 입력한 값이 반영되지 않고 있습니다. 기본 args 를 할당하고자 한다면 args 프로퍼티에 기본값을 명시해주시기 바랍니다.

```suggestion
export const Default: Story = {
  args: {
    isOpen: true,
    onClose: () => {},
    title:"Prompt 모달"
    content:"Prompt 모달 내용입니다."
    placeholder:"Prompt placeholder"
  },
  render: (args) => (
    <PropmptModal
      {...args}
    />
  ),
};
```

Reply:
[bd8bc00](https://github.com/woowacourse/react-modules/pull/130/commits/bd8bc0034ee9d0686c2a972387a75da770373096)

수정하였습니다! 꼼꼼히 봐주셔서 감사해요 😊